src/websockets/server.py

```diff
@@ -715,7 +715,9 @@ async def _close(self) -> None:
         if self.websockets:
             await asyncio.wait(
                 [
-                    asyncio.create_task(websocket.close(1001))
+                    websocket.close(1001)
```

Comment:
For consistency, it might be better to call `ensure_future()` for 3.6: https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task
(`create_task()` is basically the newer version of `ensure_future()`.)

Reply:
Or just always use `ensure_future()`, since this library maintains compatibility with 3.6.